bin/test_s3_gzip.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import smart_open
+
+URL = "s3://aws-publicdatasets/common-crawl/crawl-data/CC-MAIN-2014-35/\
+segments/1408500800168.29/warc/\
+CC-MAIN-20140820021320-00000-ip-10-180-136-8.ec2.internal.warc.gz"
+
+with smart_open.smart_open(URL) as fin:
```

Comment:
Doesn't look like a unit test. If it's some internal test you're using, no need for it to be in repo.

If it's meant to be a unit test, the right place for it is under `tests`, such as inside `tests/test_smart_open.py`.